tracer/test/Datadog.Trace.TestHelpers/TracerHelper.cs

```diff
@@ -23,9 +24,10 @@ public static ScopedTracer Create(
             IAgentWriter agentWriter = null,
             ITraceSampler sampler = null,
             IScopeManager scopeManager = null,
-            IDogStatsd statsd = null)
+            IDogStatsd statsd = null,
+            DiscoveryService discoveryService = null)
```

Comment:
Thanks for adding this 🙂 

```suggestion
            IDiscoveryService discoveryService = null)
```